docs/prod-auth-first-login.md

```diff
@@ -1,77 +1,88 @@
 
 ```
-cd /home/zulip/deployments/current
-./manage.py shell
-from zerver.models import *
-r = get_realm(settings.ADMIN_DOMAIN)
-r.restricted_to_domain=False # Now anyone anywhere can login
-r.save() # save to the database
+cd deployments/current
+./manage.py generate_realm_creation_link
 ```
 
-If you realize you set `ADMIN_DOMAIN` wrong, in addition to fixing the
-value in settings.py, you will also want to do a similar manage.py
-process to set `r.domain = "newexample.com"`.  If you've already
-changed `ADMIN_DOMAIN` in settings.py, you can use
-`Realm.objects.all()` in the management shell to find the list of
-realms and pass the domain of the realm that is not "zulip.com" to
-`get_realm`.
-
-Depending what authentication backend you're planning to use, you will
-need to do some additional setup documented in the `settings.py` template:
-
-* For Google authentication, you need to follow the configuration
-  instructions around `GOOGLE_OAUTH2_CLIENT_ID` and `GOOGLE_CLIENT_ID`.
-
-* For Email authentication, you will need to follow the configuration
-  instructions for outgoing SMTP from Django.  You can use `./manage.py
-  send_test_email username@example.com` to test whether you've
-  successfully configured outgoing SMTP.
-
-You should be able to login now.  If you get an error, check
-`/var/log/zulip/errors.log` for a traceback, and consult the next
-section for advice on how to debug.  If you aren't able to figure it
-out, email zulip-help@googlegroups.com with the traceback and we'll
-try to help you out!
+Note: You always need to run `manage.py` as the Zulip user. To start an interactive
+shell as the Zulip user, use `sudo -u zulip -i`.
+
+Open the link generated with your web browser. You'll see the create realm
+(organization) page:
+
+![Image of Zulip create realm page](images/zulip-create-realm.png)
+
+Enter your email address. You should use an address that matches the domain
+with which you want this realm (organization) to be associated.
+
+Once you provide your email address, click *Create organization* and you'll be
+emailed a confirmation link.
+
+![Image of Zulip confirmation link page](images/zulip-confirmation.png)
+
+Check your email and click this link. You'll be prompted to finish setting up
+your organization and your user:
+
+![Image of Zulip ](images/zulip-create-user-and-org.png)
+
+Complete this form and you'll be ready to log in!
+
+Your user will automatically have administrator access. You will have a special
+"Administration" tab linked to from the upper-right gear menu in the Zulip app
+that lets you deactivate other users, manage streams, change the Realm
+settings, etc.
+
+![Image of Zulip admin settings page](images/zulip-admin-settings.png)
+
+## Grant administrator access
 
-You will likely want to make your own user account an admin user,
-which you can do via the following management command:
+You can make any user and administrator with the `knight` management command:
```

Comment:
Add "on the command line" here since they can do this (but not the API super user patr) from the website as well.